downstream/modules/builder/proc-installing-builder.adoc

```diff
@@ -2,20 +2,14 @@
 
 = Installing {Builder}
 
-You can install {Builder} using Red Hat Subscription Management (RHSM) to attach your {PlatformName} subscription. https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.1/html-single/red_hat_ansible_automation_platform_installation_guide/index#proc-attaching-subscriptions_planning/[Attaching your {PlatformName} subscription] allows you to access subscription-only resources necessary to install `ansible-builder`. Once you attach your subscription, the necessary repository for `ansible-builder` is automatically enabled.
+You can install {Builder} using Red Hat Subscription Management (RHSM) to attach your {PlatformName} subscription. https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html-single/red_hat_ansible_automation_platform_planning_guide/index#proc-attaching-subscriptions_planning[Attaching your {PlatformName} subscription] allows you to access subscription-only resources necessary to install `ansible-builder`. Once you attach your subscription, the necessary repository for `ansible-builder` is automatically enabled.
 
 NOTE: You must have valid subscriptions attached on the host before installing `ansible-builder`.
```

Comment:
Should the syntax for this note be updated to the following format: 

[NOTE]
====
Text for note.
====

Even though it says so in the RH Supplementary Guide (under Structure/Admonitions, following the content under Tip), I've seen it both ways. Will leave it to you.

Reply:
LGTM w/ just one question/comment.